samples/cpp/benchmark_app/main.cpp

```diff
@@ -250,9 +250,8 @@ void fuse_mean_scale(ov::preprocess::PrePostProcessor& preproc, const benchmark_
  */
 int main(int argc, char* argv[]) {
     std::shared_ptr<StatisticsReport> statistics;
+    ov::CompiledModel compiledModel;
```

Comment:
why is it required?
have you changed a behavior ?

Reply:
Compiled model to be kept in try...catch block. The exception to be caught and processed for "[GPU] CL_OUT_OF_RESOURCES exception". In this case exit -1 to be done on application level in order to avoid consequent crashes

Reply:
Reverted